new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -359,6 +360,16 @@ public static boolean deleteConfigMap(String name, String namespace) {
     return ConfigMap.delete(name, namespace);
   }
 
+  /**
+  * List Kubernetes Config Map in a namespace.
```

Comment:
plural

Reply:
/**
  * List Kubernetes Config Map(s) in a namespace.
  * @param namespace name of namespace
  * @return List of Config Map(s) in a namespace
  */